src/fn/upload-custom-translations.js

```diff
@@ -21,11 +23,18 @@ module.exports = (projectDir, couchUrl) => {
       return Promise.all(fs.readdir(dir)
         .filter(name => FILE_MATCHER.test(name))
         .map(fileName => {
+          const id = idFor(fileName);
+          const language_code = id.substring(id.indexOf('-') + 1);
+          const language_name = ISO639.getName(language_code);
+          if (!language_name){
```

Comment:
What if we are servicing a region that is not within the ISO639 standard? Is it guaranteed to include all languages? I love the attempt to automatically set smart defaults, but I don't think we should block if the language is unknown.

Reply:
ISO 639 is not region specific and is only concerned with representation of languages and language groups. If a language isn't included in the standard, it's likely that a locale implementation of the language doesn't exist and it would be good to inform the user so that they can manage their expectations. There are many problems that can arise from introducing this much rigidity and we now warn the user when uploading an unknown language.